src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -1719,6 +1720,13 @@ public String getVariableNamePattern() {
 					})
 					.serializer(new EnumSerializer<>(Cat.Type.class)));
 		}
+
+		Classes.registerClass(new ClassInfo<>(PersistentDataHolder.class, "persistentdataholder")
```

Comment:
This fails on versions that PersistentDataHolder is not available.

Reply:
Forgot about the class info... woops